src/services/login-handler.js

```diff
@@ -0,0 +1,89 @@
+const logger = require('../services/logger.js');
+const jwt = require('jsonwebtoken');
+const UserSecretCreator = require('./user-secret-creator');
+const AuthorizationFinder = require('./authorization-finder');
+let GoogleAuthorizationFinder = require('../services/google-authorization-finder');
+
+function LoginHandler({
+  renderingId,
+  envSecret,
+  authData,
+  useGoogleAuthentication,
+  authSecret,
+  dependencies
+}) {
+  if (dependencies.GoogleAuthorizationFinder) {
+    GoogleAuthorizationFinder = dependencies.GoogleAuthorizationFinder;
+  }
+
+  const { forestToken, email, password } = authData;
+
+  function getTwoFactorResponse(user) {
+    const twoFactorSecretSalt = process.env['2FA_SECRET_SALT'];
```

Comment:
Capital letters for constants makes it easier to see that this variable is "special".

Reply:
`2FA_SECRET_SALT` to prefix with `FOREST`